project/Docker.scala

```diff
@@ -0,0 +1,29 @@
+import com.typesafe.sbt.packager.Keys._
+import com.typesafe.sbt.packager.docker.DockerPlugin.autoImport.{Docker => docker}
+import sbt.Def
+import sbt.Keys._
+
+object Docker {
+
+  lazy val dockerSettings = Seq(
+    packageName in docker := packageName.value,
+    dockerBaseImage := "anapsix/alpine-java:8",
```

Comment:
wow didn't even know about Docker store, yes that is! I'll make the change to an `openjdk` image and Olly's suggestion of a specific tag, unless anyone comes in with an objection.